build.Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN go get github.com/Masterminds/glide \
 && go get github.com/kisielk/errcheck
 
 # Which docker version to test on
-ENV DOCKER_VERSION 1.10.1
+ENV DOCKER_VERSION 1.9.1
```

Comment:
This is to make sure it works but I wonder if I should keep it or not in the end :angel: